packages/charts/react-charts-preview/library/src/components/CommonComponents/useCartesianChartStyles.styles.ts

```diff
@@ -58,21 +58,19 @@ const useStyles = makeStyles({
     '& text': {
       fill: tokens.colorNeutralForeground1,
       ...typographyStyles.caption2Strong,
-      '& selectors': {
-        [HighContrastSelectorBlack]: {
-          fill: 'rgb(179, 179, 179)',
-        },
+      '@media (forced-colors: active)': {
```

Comment:
why change css selector. Is the previous high contrast selector not working?

Reply:
For 1 scenario it was not working, checking if that exists within these changes.